hub/core/meta/tensor_meta.py

```diff
@@ -17,18 +17,38 @@ def read_tensor_meta(key: str, storage: StorageProvider) -> dict:
     return pickle.loads(storage[get_tensor_meta_key(key)])
 
 
-def tensor_meta_from_array(
-    array: np.ndarray, batched: bool, chunk_size: int = DEFAULT_CHUNK_SIZE
+def default_tensor_meta(
+    htype: Optional[str] = None,
+    chunk_size: Optional[int] = None,
+    dtype: Optional[str] = None,
+    extra_meta: Optional[dict] = None,
+):
+    if chunk_size is None:
+        chunk_size = DEFAULT_CHUNK_SIZE
+    if dtype is None:
+        dtype = DEFAULT_DTYPE
+    if extra_meta is None:
+        extra_meta = {}
+
+    tensor_meta = extra_meta
+    tensor_meta["chunk_size"] = chunk_size
+    tensor_meta["dtype"] = dtype
+    tensor_meta["length"] = 0
+    if htype is not None:
+        tensor_meta["htype"] = htype  # TODO: identify presets
```

Comment:
raise Exception for NonExistentHType or something

Reply:
@McCrearyD I don't think we want this to be the case. We don't have the full list of supported htypes yet, and it's perfectly fine to not give any at all (and just store a plain tensor).

Reply:
yeah but right now if i say `ds.create_tensor("tensor", type="imahes")` (typo) this should fail

Reply:
or right now since no presets exist, it would be confusing if i did `ds.create_tensor("tensor", htype="bbox")` and it ran perfectly fine, despite bbox not being implemented as an htype yet

Reply:
When we implement one, we can add this check as well.